app/reducers/views/channel.js

```diff
@@ -348,6 +349,44 @@ function lastChannelViewTime(state = {}, action) {
     }
 }
 
+function keepChannelIdAsUnread(state = null, action) {
```

Comment:
I find myself stumbling over this name, since it sounds like an action. Maybe `keptChannelIdAsUnread`?

Reply:
I could change it, but it is exactly the same is the one in the webapp, so I prefer to keep both the same, at least for now